components/ResearchHubPopover.tsx

```diff
@@ -49,7 +49,15 @@ export default function ResearchHubPopover({
             {popoverContent}
           </ArrowContainer>
         ) : (
-          popoverContent
+          <div
```

Comment:
I'd just check that this styling didn't mess up places where we use this (i.e. Notebook)